Api/Filter/FilterInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the DunglasJsonLdApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\JsonLdApiBundle\Api\Filter;
+
+/**
+ * Filters applicable on a resource.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+interface FilterInterface
+{
+    const STRATEGY_EXACT = 'exact';
```

Comment:
The idea was:
- `exact` is the default and must always be implemented, for each data provider
- in term of DX, it eases filter services definition in `services.yml` (optional parameter)

But a better alternative is probably better: changing the constructor prototype with `$strategy = null`.

For the `applyFilter` logic, it's something I was thinking about. I'll submit something soon.